torchio/transforms/augmentation/intensity/random_labels_to_image.py

```diff
@@ -0,0 +1,256 @@
+        gaussian_parameters: Dictionary containing the mean and standard deviation for
+            each label. For each value :math:`v`, if a tuple
+            :math:`(a, b)` is provided then
+            :math:`v \sim \mathcal{U}(a, b)`.
+            If no value is given for a label, value from default_gaussian_parameters
+            will be used.
+        default_gaussian_parameters: Dictionary containing the default
+            mean and standard deviation used for all labels that are not
+            defined in gaussian_parameters.
+            Default values are :math:`(0.1, 0.9)` for the mean and
+            :math:`(0.01, 0.1)` for the standard deviation.
+        binarize: Boolean to tell if PV label maps should be binarized.
+            Does not have any effects if not using PV label maps.
+            Binarization is done taking the highest value per voxel
+            in the different PV label maps.
+        p: Probability that this transform will be applied.
+        seed: See :py:class:`~torchio.transforms.augmentation.RandomTransform`.
+
+    .. note:: Generated images using label maps are unrealistic, therefore it is recommended
+        to blur the new images. See :py:class:`~torchio.transforms.augmentation.RandomBlur`.
+
+    Example:
+        >>> import torchio
+        >>> from torchio import RandomLabelsToImage, DATA, RescaleIntensity, Compose
+        >>> from torchio.datasets import Colin27
+        >>> colin = Colin27(2008)
+        >>> # Using the default gaussian_parameters
+        >>> transform = RandomLabelsToImage(label_key='cls')
+        >>> # Using custom gaussian_parameters
+        >>> label_values = colin['cls'][DATA].unique().round()
+        >>> gaussian_parameters = {
+        >>>     int(label): {
+        >>>         'mean': i / len(label_values), 'std': 0.01
+        >>>     } for i, label in enumerate(label_values)
+        >>> }
+        >>> transform = RandomLabelsToImage(label_key='cls', gaussian_parameters=gaussian_parameters)
+        >>> transformed = transform(colin)  # colin has a new key 'image' with the simulated image
+        >>> # Filling holes of the simulated image with the original T1 image
+        >>> rescale_transform = RescaleIntensity((0, 1), (1, 99))   # Rescale intensity before filling holes
+        >>> simulation_transform = RandomLabelsToImage(
+        >>>     label_key='cls',
+        >>>     image_key='t1',
+        >>>     gaussian_parameters={0: {'mean': 0, 'std': 0}}
+        >>> )
+        >>> transform = Compose([rescale_transform, simulation_transform])
+        >>> transformed = transform(colin)  # colin's key 't1' has been replaced with the simulated image
+    """
+    def __init__(
+            self,
+            label_key: Optional[str] = None,
+            pv_label_keys: Optional[Sequence[str]] = None,
+            image_key: str = 'image',
+            gaussian_parameters: Optional[Dict[Union[str, TypeNumber], Dict[str, TypeRangeFloat]]] = None,
```

Comment:
Can you please stick to 80-character lines? In this case, you can maybe define this Type at the top of the module.